samples/linecounter/LineCounter.csproj

```diff
@@ -13,4 +13,7 @@
   <ItemGroup>
     <Folder Include="Connected Services\" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\sdk\eventgrid\Azure.Messaging.EventGrid\src\Azure.Messaging.EventGrid.csproj" />
```

Comment:
I think we should use the package reference in samples when possible so they are independent of the repo.

Reply:
Yeah, I will update once we next release Event Grid. I can't do it yet as I am depending on the changes I have made to include the builder extensions.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/17138